README.md

```diff
@@ -3,8 +3,16 @@
 # Project Name
 Port-folio
 
-> Description the project.
-This is my first Microverse project saying Hello.
+> ## Description
+  Hello Microverse
+ In this port-folio, I:
+ - Created a master branch;
+ - Added an SSH keys to Github;
+ - Added a HTML file;
+ - Added a CSS file;
+ - Added a Linters file;
+ - Added a h1 text of "Hello Microverse";
+ - Added a styling.
```

Comment:
You are currently using the old readme file. Kindly update it with the new [readme template](https://github.com/microverseinc/readme-template). :wink: Please do not forget to edit and update the content according to your project and credentials. :pray: